osu.Game/Overlays/Dashboard/CurrentlyOnlineDisplay.cs

```diff
@@ -271,8 +270,8 @@ private void load()
                                 Anchor = Anchor.TopCentre,
                                 Origin = Anchor.TopCentre,
                                 // this is SHOCKING
```

Comment:
Probably remove this comment, because the shocking part is what you're removing here

Reply:
I considered this and left it in because it's still pretty shocking. See https://github.com/ppy/osu/pull/31524 which removes it properly.